app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,108 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+import { pick } from 'lodash-es';
+
+export default Component.extend(FormMixin, {
+
+  async didInsertElement() {
+    this._super(...arguments);
+    let actualUser = await this.authManager.currentUser;
```

Comment:
Not required ... directly use `this.authManager.currentUser`. No await needed either

Reply:
@niranjan94 Done!